service/resources/google_sheets.py

```diff
@@ -290,7 +290,51 @@ def create_spreadsheets_json(worksheet_title=None):
         {'path':['data', 'newBuildingUse', 'apartments']},
         {'path':['data', 'newBuildingUse', 'office']},
         {'path':['data', 'newBuildingUse', 'other']},
-        {'path':['data', 'newBuildingUseOther']}
+        {'path':['data', 'newBuildingUseOther']},
+        {'path':['data', 'applicantOrganizationName']},
+        {'path':['data', 'ownerOrganizationName']},
+        {'path':['data', 'ownerFirstName']},
+        {'path':['data', 'ownerLasttName']},
```

Comment:
should `ownerLasttName` be `ownerLastName`?

Reply:
nice catch.  thanks!